common/api-review/storage.api.md

```diff
@@ -35,85 +55,110 @@ export interface ListOptions extends types.ListOptions {
 
 // @public
 export interface ListResult {
-    // (undocumented)
     items: StorageReference[];
-    // (undocumented)
     nextPageToken?: string;
-    // (undocumented)
     prefixes: StorageReference[];
 }
 
 // @public
 export interface Metadata extends types.FullMetadata {
     // (undocumented)
     [prop: string]: unknown;
-    // (undocumented)
     cacheControl: string | undefined;
-    // (undocumented)
     contentDisposition: string | undefined;
-    // (undocumented)
     contentEncoding: string | undefined;
-    // (undocumented)
     contentLanguage: string | undefined;
-    // (undocumented)
     contentType: string | undefined;
-    // (undocumented)
     customMetadata: {
         [key: string]: string;
     } | undefined;
-    // (undocumented)
     downloadTokens: string[] | undefined;
-    // (undocumented)
     md5Hash: string | undefined;
-    // (undocumented)
     ref: StorageReference | undefined;
-    // (undocumented)
+    // @internal (undocumented)

```

Comment:
FWIW, this property doesn't seem to be used in the SDK.

Reply:
It gets set here. I'm not sure what the backend does or doesn't do with it but I don't want to mess with it.

https://github.com/firebase/firebase-js-sdk/blob/b95734c175ce87dc704e55466f852601b972f799/packages/storage/src/implementation/metadata.ts#L130